docs/CODEOWNERS

```diff
@@ -1,21 +1,21 @@
-# Subsystems have specific owners
-
-
-CMake/                                      @robertmaynard
-data/                                       @dpugmire
-docs/                                       @robertmaynard @kmorel
-examples/                                   @kmorel
-Utilities/                                  @kmorel
-viskores/                                       @robertmaynard @kmorel
+.github/                                        @vicentebolea
+.gitlab-ci.yml                                  @vicentebolea
+.gitlab/                                        @vicentebolea
+CMake/                                          @vicentebolea
+data/                                           @dpugmire
+docs/                                           @robertmaynard @kmorel
+examples/                                       @kmorel
+Utilities/                                      @kmorel @vicentebolea
 viskores/interop/                               @robertmaynard
 viskores/io/                                    @sujin.philip
 viskores/rendering/                             @mclarsen
```

Comment:
Uhh, we don't really have someone who knows the ins and outs of the rendering. Maybe put @nicolemarsaglia on it.